src/Tribe/Tickets.php

```diff
@@ -2130,20 +2130,14 @@ public function get_availability_slug_by_collection( $tickets, $datetime = null
 				return;
 			}
 
-			if ( is_numeric( $datetime ) ) {
-				$timestamp = $datetime;
-			} elseif ( $datetime ) {
-				$timestamp = strtotime( $datetime );
-			} else {
-				$timestamp = current_time( 'timestamp' );
-			}
-
 			$collection_availability_slug = 'available';
 			$tickets_available = false;
 			$slugs = [];
 
+			/** @var Tribe__Tickets__Ticket_Object $ticket */
```

Comment:
:no_entry_sign: Missing short description in doc comment

<sub>hash: 850e8c78c5109bd271d595930e57d45f</sub>